ow_lander/scripts/path_planning_commander.py

```diff
@@ -213,10 +213,11 @@ def stop_arm(self):
     self.trajectory_async_executer.systemFaultListener(plan.joint_trajectory, self.handle_guarded_move_feedback)
 
   def callback(self, data):
+
     """
     If system fault occurs, and it is an arm failure, path planning stops
     """
-    if data.value == 4:
+    if (data.value & 4) == 4 :
```

Comment:
Instead of magic number 4, having an enum or macro named ArmExecutionError defined to it would be clearer and more maintainable.

Reply:
sgtm!